root.c

```diff
@@ -2229,6 +2229,22 @@ static w_root_t *root_resolve(const char *filename, bool auto_watch,
     watch_path = (char*)filename;
   }
 
+  if (watch_path) {
+    struct stat st;
+    int res = w_lstat(filename, &st, true);
+
+    if (res != 0 && errno == ENOENT) {
+      ignore_result(asprintf(errmsg, "casing problem in \"%s\"", filename));
```

Comment:
Let's expand the text here, so that it is clearer about what needs to be done:

```
"\"%s\" resolved to \"%s\" but we were unable to examine \"%s\" using strict "
"case sensitive rules.  Please check each component of the path and make "
"sure that that path exactly matches the correct case of the files on your "
"filesystem.",  filename, watch_path, filename);
```